bciers/apps/registration/app/components/form/widgets/readOnly/ReadOnlyComboBoxWidget.tsx

```diff
@@ -10,6 +10,7 @@ const ReadOnlyComboBoxWidget: React.FC<WidgetProps> = ({
   ];
   const selectedValue =
     fieldSchema &&
+    value &&
```

Comment:
Found this bug while testing the empty state of the read only widget. Without this added it would show a selected value even if there was no value 😱